plugins/pulp_ostree/plugins/lib.py

```diff
@@ -203,22 +301,30 @@ def list_refs(self):
         return _list
 
     @wrapped
-    def pull(self, remote_id, refs, listener):
+    def pull(self, remote_id, refs, listener, depth=-1):
```

Comment:
Does it make sense to use `DEFAULT_DEPTH` here? I can see how that might not, but thought I'd mention it. Up to you, though.